.drone.star

```diff
@@ -2005,7 +1995,7 @@ def githubComment(earlyFail):
     if (earlyFail):
         return [{
             "name": "github-comment",
-            "image": "jmccann/drone-github-comment:1",
+            "image": THEGEEKLAB_DRONE_GITHUB_COMMENT,
             "pull": "if-not-exists",
             "settings": {
                 "message_file": "%s/comments.file" % dir["base"],
```

Comment:
this new image has a little different configuration. please refer to the changes in the activity app and fix the case here